src/main/java/net/minecraftforge/common/ForgeVersion.java

```diff
@@ -204,6 +209,46 @@ public void run()
                 }
             }
 
+            /**
+             * Opens stream for given URL while following redirects (max. 5)
```

Comment:
Not sure if this should say 20 or if the default `MAX_HTTP_REDIRECTS` is supposed to be 5.
I think you should not write `(max 5)` since it's not really a maintainable comment.